sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+\section{Sensitivity Analysis}
+
+The most important aspect of any fuel cycle transition scenario
+is the accrual of fissile materials for new reactor deployment.
+The collaborative strategy makes a transition possible materialistically,
+but the aggressive transition demands an increase in reprocessing capacity.
+This is due to both high initial plutonium loading of the \gls{ASTRID}
+reactors and the low plutonium composition in spent \gls{UOX} fuel.
+
+We varied two parameters, the lifetime of French \glspl{LWR} and the
+breeding ratio of \gls{ASTRID} reactors. The parameters and their values
+are listed in \cref{tab:sen_par}
+
+\begin{table}[h]
+    \centering
+    \begin{tabularx}{\textwidth}{lbb}
+        \hline
+        \textbf{Parameter} & \textbf{Original} & \textbf{Values} \\
+        \hline
+        Lifetime of French \glspl{LWR} [years] & 60  & 65, 70, 80 \\
+        Breeding Ratio of \glspl{ASTRID} & 1.08 & 1.11, 1.15, 1.18 \\ 
+        \hline
+    \end{tabularx}
+    \caption {Two parameters are varied for the sensitivity analysis - the 
+              lifetime of French \glspl{LWR} and the breeding ratio of \glspl{ASTRID}.
+              The parameters are increased to analyze their impact on the reprocessing
+              demand for the transition.}
+    \label{tab:sen_par}
+\end{table}
+
+\subsection{Breeding Ratio}
+
+The result of the breeding ratio sensitivity analysis is shown in
+\cref{fig:br_rep}.
+Increase in the breeding ratio of \gls{ASTRID} reactors
+decreases the monthly average reprocessing, but the 
+average \gls{UOX} reprocessing demand doesn't dramatically change,
```

Comment:
let's use some words that are more quantitative. For example, instead of "dramatically" consider reporting what percentage the reprocessing demand changed when the parameter was varied.